gemini/use-cases/retrieval-augmented-generation/small_to_big_rag/small_to_big_rag.ipynb

```diff
@@ -0,0 +1,848 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    def get_text_embedding(doc) -&gt; list:
</pre><p>Add typing for parameters</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/485/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>